app/split_integer.py

```diff
@@ -1,7 +1,13 @@
+
```

Comment:
The function `split_integer` should handle cases where `number_of_parts` is greater than `value`. Currently, it will produce parts that do not sum up to the original `value`.